rules/renaming/tests/Rector/FuncCall/RenameFunctionAutoimportRector/config/configured_rule.php

```diff
@@ -1,15 +1,18 @@
 <?php
 
-declare(strict_types=1);
-
-use Rector\CodingStyle\Tests\Rector\Use_\UseFunction\Source\UseFunctionRector;
 use Rector\Core\Configuration\Option;
+use Rector\Renaming\Rector\FuncCall\RenameFunctionRector;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $parameters = $containerConfigurator->parameters();
     $parameters->set(Option::AUTO_IMPORT_NAMES, true);
 
     $services = $containerConfigurator->services();
-    $services->set(UseFunctionRector::class);
+    $services->set(RenameFunctionRector::class)
+        ->call('configure', [[
+            RenameFunctionRector::OLD_FUNCTION_TO_NEW_FUNCTION => [
+                'view' => 'Laravel\Templating\render',
```

Comment:
Where is the `Laravel\Templating\render` autoloaded in tests? I can't find it

Reply:
This is an old commit. First I was trying it with the same set as in existing renaming tests. But rector just change names and didn't use auto import names, so that's why I used `service` function